transformer/external/executabletransformer.go

```diff
@@ -115,8 +123,8 @@ const (
 func (t *Executable) Transform(newArtifacts []transformertypes.Artifact, alreadySeenArtifacts []transformertypes.Artifact) (pathMappings []transformertypes.PathMapping, createdArtifacts []transformertypes.Artifact, err error) {
 	pathMappings = []transformertypes.PathMapping{}
 	createdArtifacts = []transformertypes.Artifact{}
-	for _, a := range newArtifacts {
-		if t.ExecConfig.TransformCMD == nil {
+	if t.ExecConfig.TransformCMD == nil {
```

Comment:
I think it might be better to make TransformerCMD as mandatory and return error in init itself in transformcmd is not there. The default behavior will be confusing. They can achieve the default behaviour with a starlark transformer if required.

Reply:
done.